### DISCUSSION TITLE
UART ELRS Rx onboard solid green LED indicator

### DISCUSSION THREAD

**Discussion by hanggarmungil** [ExpressLRS/ExpressLRS#2895] [Q&A]
I have issue with My Happymodel Crux35 ELRS V2 / Crux35 HDZERO flight control. Since I have received the ELRS receiver led indicator give solid green. And I can't bind the FC to my ELRS Tx. I have follow manual to activated UART2 for serial Rx and UART1 activated for configuration/MSP. My Tx is newest firmware ELRS V3. I also folIow bind instruction by 3 time plug-unplug USB, but nothing happened. I updated betaflight firmware to 4.5.1 with target Crazybeef4dx. But the green led indicator still solid green.
Just for information, the HD Vtx and DJI Goggles V2 already success bound (with only low power, 25mw). 
I have also check ELRS boot button on FC, and I think not on press/push condition.
How to solve the FC problem so I can bind my ELRS Tx and onboard Rx.

https://github.com/user-attachments/assets/47441f0b-b588-4b96-ae0e-418814bb1b16

Regards,

**Comment by GriVoran** [comment]
Did you ever solve this? I have the same problem. Correct ELRS firmware flashed but led is solid green. Cant bind and cant go into wifi mode.

**Reply by hanggarmungil** [reply]
I have solved the problem, try to update internal elrs firmware.
Follow the update UART internal ELRS Rx by:
push UART ELRS boot button,
plug lipo
![Screenshot 2024-09-15 202702](https://github.com/user-attachments/assets/988be8fc-fb39-49e8-831c-f4f1626fa5f4)
next follow the page (https://www.expresslrs.org/quick-start/receivers/updating/)

**Reply by GriVoran** [reply]
Thank you for your reply. I just did exactly this and it worked. I did the passthrough update using thr bootloader button. Now everything is working correctly. Thanks again.

**Reply by hanggarmungil** [reply]
Glad to help. Congratulation bro. Let's Fly.
